website/content/docs/enterprise/license/faq.mdx

```diff
@@ -31,6 +32,35 @@ This FAQ section is for the license changes introduced in Vault Enterprise 1.11.
 
 Per the [feature deprecation plans](/docs/deprecation), Vault will no longer support licenses in storage. An [auto-loaded license](/docs/enterprise/license/autoloading) must be used instead. If you are using stored licenses, you must migrate to auto-loaded licenses prior to upgrading to Vault 1.11
 
+Vault 1.12 will also introduce different termination behavior for evaluation licenses versus non-evaluation licenses. An [evaluation](https://www.hashicorp.com/products/vault/trial) license will include a 30-day trial period after which a running Vault server will terminate. Vault servers using a non-evaluation license will not terminate.
+
+### Q: How do the license termination changes affect upgrades?
+Vault 1.12 will introduce changes to license termination behavior. Upgrades when using expired licenses will now be limited.
+Vault will not startup if the build date of the binary is _after_ the expiration date of a license. License expiration date and binary build date compatibility can be verified using the [Check for Autoloaded License](/docs/commands/operator/diagnose#check-for-autoloaded-license) check performed by the `vault operator diagnose` command.
+The build date of a binary can also be found using the [vault version](/docs/commands/version#version) command.
+
+A user can expect the following to occur based on the following scenarios:
+
+**Evaluation or non-evaluation license is valid:**
+Vault will start normally
+
+**Evaluation or non-evaluation license is expired, binary build date _before_ license expiry date:**
+Vault will start normally
+
+**Evaluation or non-evaluation license is expired, binary build date _after_ license expiry date:**
+Vault will not start
+
+**Evaluation license is terminated:**
+Vault will not start independent of the binary’s build date
+
+**Non-evaluation license is terminated, binary build date _before_ license expiry date:**
+Vault will start normally
+
+**Non-evaluation license is terminated, binary build date _after_ license expiry date:**
+Vault will not start
+
+The Vault support team will issue you a temporary evaluation license to allow for security upgrades if your license has expired.
```

Comment:
Should this be 'can issue you'? Will we pro-actively reach out or do they come to us and ask?